src/include/daos_srv/daos_mgmt_srv.h

```diff
@@ -16,16 +16,8 @@
 
 #include <uuid/uuid.h>
```

Comment:
`<uuid/uuid.h>` has been included via `<daos_srv/daos_mgmt_srv_file.h>`.

Reply:
`<daos_srv/daos_mgmt_srv_file.h>` is included here for convenience so a bunch of files do not have to include this new header. But it does not make these two somehow fused together.

`<daos_srv/daos_mgmt_srv_file.h>` is not included for `<uuid/uuid.h>` and `uuid_t` is still used extensively across `daos_mgmt_srv.h` directly. This is why I think it is ok to keep both.